scripts/download_openmeteo_2024_2025.py

```
@@ -0,0 +1,151 @@
+    
+    if successful:
+        print("\n✓ Successful downloads:")
+        for month in successful:
+            print(f"  - {month}")
+    
+    if failed:
+        print("\n✗ Failed downloads:")
+        for month in failed:
+            print(f"  - {month}")
+        
+        print(f"\nTo retry failed months individually:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)